devito/passes/iet/misc.py

```diff
@@ -270,3 +306,39 @@ def _rename_subdims(target, dimensions):
     return {d: d._rebuild(d.root.name) for d in dims
             if d.root not in dimensions
             and names.count(d.root.name) < 2}
+
+
+_stdcomplex_defs = """
```

Comment:
imho this belongs to a `complex.h`

Reply:
actually, given all the other comments (this is the final one I'm writing), you may as well move the entire complex number lowering machinery to a separate python module such as `complex.py` within `passes/iet/`

Reply:
>imho this belongs to a complex.h

This is a lot more robust to generate the header in the same dir as the the generated code and avoid having to infer path from the devito dir. 

> complex.py

That's fine